code/game/objects/items/toys.dm

```diff
@@ -904,8 +909,248 @@
+			on = !on
+			update_brightness()
+	else
+		atom_say("Self-destruct command received!</span>")
+		visible_message("<span class='danger'>[src] explodes!</span>")
+		var/turf/T = get_turf(src)
+		playsound(T, 'sound/goonstation/effects/robogib.ogg', 50, 1)
+		robogibs(T)
+		if(grenade)
+			explosive_betrayal(grenade)
+		if(!QDELETED(src))
+			qdel(src)
+
+/obj/item/toy/plushie/dionaplushie
+	name = "diona plushie"
+	desc = "This plushy is seemingly comprised of other, smaller, nymph plushies. They really went all out on the realism! Keep away from plantkiller."
+	icon_state = "plushie_diona"
+	COOLDOWN_DECLARE(creak_cooldown)
+
+/obj/item/toy/plushie/dionaplushie/attack_self(mob/user)
+	if(!COOLDOWN_FINISHED(src, creak_cooldown))
+		return ..()
+	playsound(src,'sound/voice/dionatalk1.ogg', 10, 0)
+	visible_message("<span class='danger'>Creak...</span>")
+	COOLDOWN_START(src, creak_cooldown, 3 SECONDS)
+	return
+
+/obj/item/toy/plushie/nymphplushie
+	name = "nymph plushie"
+	desc = "Life-sized plushie of a diona nymph, perhaps if you find another you could make a diona!"
+	icon_state = "plushie_nymph"
+	COOLDOWN_DECLARE(chirp_cooldown)
+
+/obj/item/toy/plushie/nymphplushie/attackby(obj/item/B, mob/user, params)
+	if(istype(B, /obj/item/toy/plushie/nymphplushie))
+		var/obj/item/toy/plushie/nymphplushie/NP = B
+		var/found_grenade = FALSE
+		if(grenade)
+			found_grenade = TRUE
+			explosive_betrayal(grenade)
+		if(NP.grenade)
+			found_grenade = TRUE
+			NP.explosive_betrayal(NP.grenade)
+		if(found_grenade)
+			return
+		new /obj/item/toy/plushie/dionaplushie(get_turf(loc))
+		to_chat(user, "<span class='notice'>The nymph plushies combine seamlessly into an diona plushie!</span>")
+		playsound(loc, 'sound/voice/dionatalk1.ogg', 50, 1)
+		qdel(NP)
+		qdel(src)
+	else
+		return ..()
+
+/obj/item/toy/plushie/nymphplushie/attack_self(mob/user)
+	if(!COOLDOWN_FINISHED(src, chirp_cooldown))
+		return ..()
+	playsound(src,'sound/creatures/nymphchirp.ogg', 10, 0)
+	visible_message("<span class='danger'>Chirp!</span>")
+	COOLDOWN_START(src, chirp_cooldown, 3 SECONDS)
+	return
+
+/obj/item/toy/plushie/plasmamanplushie
+	name = "plasmaman plushie"
+	desc = "A freindly plasma-being in plush form. WARNING: KEEP AWAY FROM OPEN FLAME!"
+	icon_state = "plushie_plasma"
+	COOLDOWN_DECLARE(rattle_cooldown)
+
+/obj/item/toy/plushie/plasmamanplushie/welder_act(mob/user, obj/item/I)
+	if(I.use_tool(src, user, volume = I.tool_volume))
+		bakoom()
+	return TRUE
+
+/obj/item/toy/plushie/plasmamanplushie/attackby(obj/item/I, mob/living/user, params)
+	if(I.get_heat())
+		bakoom()
+	else
+		return ..()
+
+/obj/item/toy/plushie/plasmamanplushie/fire_act(datum/gas_mixture/air, exposed_temperature, exposed_volume, global_overlay = TRUE)
+	..()
+	bakoom()
+
+/obj/item/toy/plushie/plasmamanplushie/proc/bakoom()
+	visible_message("<span class='danger'>[src] explodes!</span>")
+	if(grenade)
+		explosive_betrayal(grenade)
+	explosion(get_turf(src), -1, 0, 1, 1, flame_range = 1)
```

Comment:
Personally I dont think this is a good idea, but I wont be the one to stop you

Reply:
This only effects the tile the person is on, and would require them to weld it in their hand or next to them, I'm confident this wouldn't be a very effective griefing tool in the least.